runtime/extension/extension_helper.cpp

```diff
@@ -46,8 +48,13 @@ uint64_t bpftime_path_join(const char *filename1, const char *filename2,
 
 namespace bpftime
 {
-#ifdef BPFTIME_ENABLE_IOURING_EXT
+/*
+io_uring are only available in linux atm 
+so adding linux guards to the following code
+*/
 
+#ifdef BPFTIME_ENABLE_IOURING_EXT
```

Comment:
Nested `#ifdef - #if` could be replaced with `#if defined(XXX) && YYY` to make it clearer